scene/gui/color_picker.cpp

```diff
@@ -2214,8 +2223,26 @@ ColorPicker::ColorPicker() {
 	btn_add_preset->set_accessibility_name(ETR("Add Preset"));
 	btn_add_preset->connect(SceneStringName(pressed), callable_mp(this, &ColorPicker::_add_preset_pressed));
 	preset_container->add_child(btn_add_preset);
+
+#ifdef MACOS_ENABLED
+	perm_hb = memnew(HBoxContainer);
+	LinkButton *perm_link = memnew(LinkButton);
+	perm_link->set_text(ETR("Screen Recording permission missing!"));
+	perm_link->set_tooltip_text(ETR("Screen Recording permission is required to pick colors from the screen.\nClick here to request access..."));
```

Comment:
This is a bit misleading, because you _can_ pick colors, it's just restricted.

Reply:
Changed "screen" to "other application windows".